src/components/toggle/toggle.stories.tsx

```diff
@@ -33,26 +28,5 @@ export const DefaultStory = (args: ToggleProps) => {
 DefaultStory.args = {
   checked: true,
   label: 'I am a label',
-};
-
-const TooltippedIcon = Tooltip(Icon);
-export const withTooltip = () => {
-  const [checked, setChecked] = useState(false);
-  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
-    setChecked(event.currentTarget.checked);
-  };
-  return (
-    <Toggle
-      checked={checked}
-      onChange={handleChange}
-      label={
-        <Box display="flex">
-          Label
-          <TooltippedIcon tooltip={<TextSmall>Tooltip</TextSmall>} marginLeft={2}>
-            <IconInfoBadgedSmallFilled />
-          </TooltippedIcon>
-        </Box>
-      }
-    />
-  );
+  tooltip: 'This is the label tooltip text',
```

Comment:
Q: Why is the tooltip added to the default story?

Reply:
Because storybook controls set is as object, where when you want to edit it, it throws error and whole page crashes. Setting default value prevent this.

Reply:
Got it 👍 Can we have the extra story without the tooltip then?

Reply:
I dont see much of a point having another story without `tooltip` when user can control main story. In that case we should create story for each prop.

Reply:
The point is that there were two stories in the beginning. And we always do showcase some of the important use cases in the stories. That is the whole point of stories actually 😅

Reply:
I did some research and you are right, we should do more stories to show different cases, generally in whole library. Thanks for bringing that up.